reviewguide/review_points_guide.md

```diff
@@ -0,0 +1,221 @@
+# Review points in Guide
+
+This page describes the review points that were often asked to be updated.
+
+When you review a guide page, check these points and get the page consistency.
+
+- [Headings](#headings)
+- [Created page](#create)
+   - [toc_all.md](#create_toc_all)
+   - [overview.md](#create_overview)
+- [Changed file name / delete the page](#change_delete)
+- [Code block](#codeblock)
+   - [Indentation of the code block](#codeblock_inden)
+   - [Code block type](#codeblock_type)
+   - [Consistency between code block and description](#codeblock_check)
+- [Hyper link](#hyperlink)
+   - [Page in docs.tizen.org](#hyperlink_guide)
+   - [Native or Web API](#hyperlink_not_dotnet)
+   - [.NET API](#hyperlink_dotnet)
+   - [Outside of docs.tizen.org](#hyperlink_out)
+- [Images](#images)
+- [Related information](#related_info)
+   - [Dependencies](#related_info_dependencies)
+

```

Comment:
How about mentioning the check for Note Tag?

Reply:
I thought the style guide will work for it. I added the link to that part.